users/models.py

```diff
@@ -71,6 +71,22 @@ def __str__(self):
         return self.email
 
 
+class Preferences(models.Model):
+
```

Comment:
This should probably have the OneToOne defined here and not Apartment model. This table needs  some identifier of either Owner or Apartment to make any sense.

Reply:
Is it connected to that Apartment or to the Owner of the apartment?
You currently have Owner as a foreign key, which means that the same user can be the owner in multiple apartments.
However, if the preference is on the person, then the OneToOne should be on the User, and then it works for all types of users: both Apartments and Seekers

Reply:
The problem that we are facing with is that Preferences need to be connect to Apartment and Seeker(in a future PR) as well. So I think that if we put the one-to-one field in Preferences it would create a problem.
There is another way to deal with that scenario?

Reply:
I think that Preferences should define details about each apartment so it better to connect it to the Apartment model.

Reply:
The problem is from a structural perspective, there is no identifying mark on the Perspective table and the OneToOne Relationship is indicating the Preferences is the primary record. 
In other words, if you create a Preference record and don't attach it to a specific Apartment, it has absolutely no meaning. Also if you delete the PreferenceID in the Apartment table, the Preferences Record still exists, but it isn't connected to anything.

Reply:
I did some changes to the code structure,  I think that it will do the best job for our needs.